docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+khronos_profiles.default_feature_values = DEFAULT_FEATURE_VALUES_DEVICE
+
+# Debug Actions
+# =====================
+# This indicates what action is to be taken when a layer wants to report
+# information
+khronos_profiles.debug_actions = DEBUG_ACTION_STDOUT_BIT
+
+# Message Types
+# =====================
+# This is a comma-delineated list of options telling the layer what types of
+# messages it should report back
+khronos_profiles.debug_reports = DEBUG_REPORT_WARNING_BIT,DEBUG_REPORT_ERROR_BIT,DEBUG_REPORT_NOTIFICATION_BIT
+```
+
+#### Layer Settings File location on Linux and macOS
+
+Unix systems store the layer setting file in the following path:
+
+- `$HOME/.local/share/vulkan/settings.d/vk_layer_settings.txt`
+
+#### Layer Settings File location on Windows
+
+Windows systems store the layer setting file in the following path:
+
+- `%HOME%\AppData\Local\LunarG\vkconfig\override\vk_layer_settings.txt`
+
+## Configuring Vulkan Layers using Environment Variables
+
+### Finding Vulkan Layers
+
+In order to enable a Vulkan layer from the command-line, you must first make sure:
+
+  1. The layer's Manifest JSON file is found by the Vulkan Desktop Loader because it is in:
+      * One of the standard operating system install paths
+      * It was added using one of the layer path environment variables (`VK_LAYER_PATH` or `VK_ADD_LAYER_PATH`).
+      * See the `Layer Discovery` section of the Vulkan Loader's [Layer Interface doc](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/docs/LoaderLayerInterface.md).
+  2. The layer's library file is able to be loaded by the Vulkan Desktop Loader because it is in:
+      * A standard library path for the operating system
+      * The library path has been updated using an operating system-specific mechanism such as:
+          * Linux: adding the path to the layer's library .so with `LD_LIBRARY_PATH`
+          * MacOS: adding the path to the layer's library .dylib with `DYLD_LIBRARY_PATH` 
+  3. The layer's library file is compiled for the same target and bitdepth (32 vs 64) as the application
+
+#### Activating Specific SDK Layers
+
+To activate layers located in a particular SDK installation, or layers built locally from source, specify the layer JSON
+manifest file directory using either `VK_LAYER_PATH` or `VK_ADD_LAYER_PATH`.
+The difference between `VK_LAYER_PATH` and `VK_ADD_LAYER_PATH` is that `VK_LAYER_PATH` overrides the system layer paths
+so that no system layers are loaded by default unless their path is added to the environment variable.
+`VK_ADD_LAYER_PATH` on the otherhand, causes the loader to search the additional layer paths listed in the
+environment variable first, and then the standard system paths will be searched.
+
+#### Example Usage On Windows:
+
+For example, if a Vulkan SDK is installed in `C:\VulkanSDK\1.2.198.0`, execute the following in a Command Window:
+
+```
+C:\> set VK_LAYER_PATH=C:\VulkanSDK\1.2.198.0\Bin
```

Comment:
This is a really old SDK. Worth updating to latest stable.

Reply:
Update...